### ISSUE TITLE
Unable to use `schemas` actions when long query is running

### ISSUE THREAD

**Issue by ThePrez** [halcyon-tech/vscode-db2i#47]
![image](https://github.com/halcyon-tech/vscode-db2i/assets/17914061/dcb8bf1b-4447-4ba2-8209-272e22d45370)

**Comment by worksofliam** [comment]
@ThePrez Does this get solved since we merged #43?

**Comment by ThePrez** [comment]
@worksofliam yes, I did some basic testing and it seems to be fixed by #43. 

FYI, "long running queries" used in my test were
```sql
SELECT * FROM TABLE(QSYS2.RECEIVE_DATA_QUEUE(
                                        DATA_QUEUE => 'BASHQ', 
                                        DATA_QUEUE_LIBRARY => 'JESSEG'))
```
```sql
select * from qsys2.syscolumns
```
